experiments/ClimaEarth/run_amip.jl

```diff
@@ -275,6 +275,16 @@ if mode_name == "amip"
         energy_check = energy_check,
     )
 
+    # Update land surface temperature to account for orography
+    surface_elevation = CC.Fields.field_values(
+        CC.Fields.level(CC.Fields.coordinate_field(atmos_sim.integrator.u.f).z, CC.Utilities.half),
+    )
+    T_sfc = Interfacer.get_field(land_sim, Val(:surface_temperature))
+    orog_adjusted_T_sfc = parent(T_sfc) .- FT(6.5e-3) .* parent(surface_elevation)
```

Comment:
would this work without parent?
```suggestion
    orog_adjusted_T_sfc = T_sfc .- FT(6.5e-3) .* surface_elevation
```

Reply:
also, could you add a comment explaining where `6.5e-3` comes from?

Reply:
Added a new variable explaining this number

Reply:
I think parent is needed here because of the type / instance issue